src/app/functions/server/dashboard.ts

```diff
@@ -65,16 +65,14 @@ export interface DashboardSummary {
   fixedDataPoints: DataPoints;
   /** manually resolved data broker data points separated by data classes */
   manuallyResolvedDataBrokerDataPoints: DataPoints;
-  /** in-progress & resolved/removed data points separated by data classes */
-  inProgressFixedDataPoints: DataPoints;
 
   /** sanitized all data points for frontend display */
   unresolvedSanitizedDataPoints: SanitizedDataPoints;
-  /** sanitized resolved/removed data points for frontend display */
-  inProgressFixedSanitizedDataPoints: SanitizedDataPoints;
+  /** sanitized resolved and removed data points for frontend display */
+  fixedSanitizedDataPoints: SanitizedDataPoints;
```

Comment:
The essence of the changes in this PR is:
1. Renamed `inProgressFixedSanitizedDataPoints` to `fixedSanitizedDataPoints`.